index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- fonts links -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,100;1,300;1,400&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+    <title>Dbn Hiphop Fest</title>
+    
+</head>
+<body>
+    <header id="header-section">
+    <nav class="top-nav-bar">
+        <img src="images/fb.png" alt="facebook-icon">
+        <img class="twitter" src="images/twitter.png" alt="Twitter-icon">
+        <ul class="top-bar">
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+
+        </ul>
+    </nav>
+          <a class="Logo logo1" href="#">Techmoves</a>
+          <a class="Logo10" href="#">Techmoves</a>
+        <nav>
+     <div class="mobile-menu">       
+            <img class="humburger" src="images/hum1.png" alt="humburger">
+            <ul>
+              <li><a class="col-nav1" href="./index.html">Home</a>
+            </li>
+              <li><a class="col-nav1" href="./about.html">About</a>
+            </li>
+              <li><a href="#main-program-sec">Feature</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#">Updates</a></li>
+              <li><a class="col-nav" href="#">CC campaign</a></li>
+            </ul>
+         </div>
+        </nav>
+    </header>
+     <div class="option">
+        <button id="closebtn">&times;</button>
+        <ul>
+           <li><a class="col-nav1" href="./index.html">Home</a>
+            </li>
+            <li><a class="col-nav1" href="./about.html">About</a>
+            </li>
+            <li><a href="#main-program-sec">Feature</a></li>
+        </ul>
+    </div> 
+    <section id="top-sections">
+    <div class="notes">
+    <p class="great-text" >"Techmoves Records Brings"</p>
+   <h1>Durban hip hop <br> fest 2023</h1>
+<div class="center-main">
+<article>It time to elevete the culture of Durban Hip hop that gonna takes place on 15 of june 2023 from 11 am, til 12am at KingSpark Stadium.go get ticket at coputerticket and ticket won't be sold at the door</article>  
+ </div>
+<div class="center-main1">
+ <h2 class="date">
+    2023.06.15(SAT) ~ Till Late
+   
+</h2>
+<span class="loaction">@KingSpark Stadium</span>
+ </div>     
+ </div>
+   
+</section>
```

Comment:
- [ ] As suggested earlier, kindly ensure that there is background-image of your **choice** for this section according to the given design. **Kindly take note of the layout as well for both mobile and desktop versions.** 

> |     | Given template                                | Your Design                                   |     |
> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
> |     | <img width="214" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235307315-e69055d5-ca16-4050-97c6-f5687f9b2b17.png"> | <img width="263" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235307278-e924f21b-5b8b-4efc-92d8-4cbfcb73059b.png"> |